qmi/core/transport.py

```diff
@@ -1117,14 +1125,17 @@ def create_transport(transport_descriptor: str,
     String format:
       - TCP connection:    "tcp:host[:port][:connect_timeout=T]"
       - Serial port:       "serial:device[:baudrate=115200][:databits=8][:parity=N][:stopbits=1]"
-      - USBTMC device:     "usbtmc[:vendorid=V][:productid=P]:serialnr=S"
+      - USBTMC device:     "usbtmc:vendorid=0x[vid]:productid=0x[pid]:serialnr=[sn]"
```

Comment:
Idea was just to improve it with adding the '0x' in front of the vendor/product IDs as it is often not otherwise clear that the "0x" should precede the number - a hex number.... 

I also realized the 'gpib' transport was wrong and there were no unit-tests for testing the transport. Now added those tests.